Bottom: 24284eeea150c467c71fccd737654a160ac370a3
Top:    1735763dbb389868e501f543994d4b4e373b8174
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:13:03 +0000

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon1.c b/src/polygon1.c
index f34c968..3a389db 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -109,13 +109,13 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-//#define DEBUG_INTERSECT
+#undef DEBUG_INTERSECT
 #undef DEBUG_LABEL
 #undef DEBUG_ALL_LABELS
-//#define DEBUG_JUMP
-//#define DEBUG_GATHER
+#undef DEBUG_JUMP
+#undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-//#define DEBUG
+#undef DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
